xdsl/dialects/llvm.py

```diff
@@ -92,6 +92,29 @@ def typed(type: Attribute):
         return LLVMPointerType([type, NoneAttr()])
 
 
+
+
+@irdl_op_definition
+class GetElementPtrOp(Operation):
+    name = "llvm.getelementptr"
+    
+
+    ptr: Annotated[Operand, LLVMPointerType]
```

Comment:
I think you will need some more things here. Especially the result is certainly required as well. I am fine with not handling the other things yet, if you don't need them...

Reply:
Just don't call it `result` or you'll have a bad time :) I suggest `res`